documentation/prometheus-mixin/dashboards.libsonnet

```diff
@@ -91,6 +91,63 @@ local g = import 'grafana-builder/grafana.libsonnet';
           { yaxes: g.yaxes('ms') } +
           g.stack,
         )
-      )
-  },
+      ),
+    # Remote write specific dashboard.
+    'prometheus-remote-write.json':
+      g.dashboard('Prometheus Remote Write')
+        .addMultiTemplate('instance', 'prometheus_build_info', 'instance')
+          .addRow(
+            g.row('Timestamps')
+            .addPanel(
+              g.panel('Highest Timestamp In vs. Highest Timestamp Sent') +
+              g.queryPanel('prometheus_remote_storage_highest_timestamp_in_seconds{instance=~"$instance"} - ignoring(queue) group_right(instance) prometheus_remote_storage_queue_highest_sent_timestamp_seconds{instance=~"$instance"}', '{{queue}}') +
```

Comment:
Prefer group_left to group_right.

Reply:
That's wouldn't work, `prometheus_remote_storage_highest_timestamp_in_seconds` doesn't have the `queue` label but we'd want the graph to display the difference in seconds for each remote write queue. I don't know promql very well so if you have a suggestion for an alternative query let me know.

Reply:
Flip the RHS and LHS

Reply:
That would give a negative value, I guess then I'd just `* -1`?

Reply:
Yeah. I partially regret adding group_right (for symmetry), this is the first case where I'd seen it kinda making sense.